components/index.html

```diff
@@ -1100,7 +1100,7 @@ <h4 class="footer__headline">Support</h4>
             <div class="col-md-6">
                 <h4 class="footer__headline">Copyright</h4>
                 <p>
-                    &copy; 2023 by <a href="https://getlaminas.org/">Laminas Project</a> a Series of LF Projects, LLC.
+                    &copy; 2024 by <a href="https://getlaminas.org/">Laminas Project</a> a Series of LF Projects, LLC.
```

Comment:
_I would kill it, but US law says something else._

Therefore my suggestion:
```suggestion
                    &copy; 2019 - present by <a href="https://getlaminas.org/">Laminas Project</a> a Series of LF Projects, LLC.
```

Reply:
Everywhere. Thanks!